src/Common/MetricHelper.cs

```diff
@@ -292,6 +292,20 @@ public void SetPSHost(PSHost host)
         {
         }
 
+        private static void PopulateAuthenticationPropertiesFromQos(AuthenticationTelemetryData telemetry, IDictionary<string, string> eventProperties)
```

Comment:
```suggestion
        private static void PopulateAuthenticationPropertiesFromQos(AzurePSQoSEvent qos, IDictionary<string, string> eventProperties)
```
because the naming of this function is ...FromQos

Reply:
Good point. Will also discuss with yeming about the naming